docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -30,13 +31,32 @@ All BPMN, DMN, and form files contained in the process application folder are de
 If any resource fails to deploy, the whole deployment [fails](#deployment-errors) and the cluster state remains unchanged. This safely ensures that a process application cannot be deployed incompletely or in an inconsistent state.
 :::
 
+## Play and validate your process application
+
+You can use [Play](/components/modeler/web-modeler/play-your-process.md) to run and validate your process application in a development cluster before deploying and running it in a testing, staging, or production cluster.
+
+To validate your process application:
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select the **Play** tab to start a Play environment using your development cluster.
```

Comment:
The concept of a "play environment" was removed in 8.6. The play environment `is` the dev cluster